src/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -45,23 +45,36 @@ class GcsPlacementGroupSchedulerInterface {
   /// Schedule the specified placement_group.
```

Comment:
Since this is now handling partial scheduling, should we rename this function and update the comment? Perhaps it can be called `ScheduleUnplacedBundles()`.

Reply:
fixed